app/views/infuse.html

```diff
@@ -5,14 +5,23 @@
     <p>Disclaimer: DIM cannot in any way infuse any of your weapons, it's impossible.</p>
     <div class="details">
       <div class="infuseSourceItem" dim-infuse-item item-data="infuse.item"></div>
-      <div class="infuseSourceDetails">
+
+      <div class="infuseSourceDetailsRow">
         <span><strong>{{infuse.item.name}}</strong></span><br />
         <span>{{infuse.item.type}}</span>
       </div>
+
       <div class="infuseSourceDetailsRow">
         <span>Attack</span><br />
         <span class="bigValue">{{infuse.item.primStat.value}}</span>
       </div>
+
+      <div class="infuseSourceDetailsRow">
+        <input class="controls" type="checkbox" ng-model="infuse.getAllItems" ng-change="infuse.getItems()"><span>Show items from every character</span>
+        <br class="clear" />
+        <input class="controls" type="checkbox" ng-model="infuse.showLockedItems" ng-change="infuse.getItems()"><span>Show locked items</span>
```

Comment:
bit rough that we need to loop all of the stores and build the item list every time we check or uncheck the box... could we instead save the items from the first getItems() and the when they check just filter that list?

maybe just a thought for a future iteration.